### PR TITLE
don't send ok message to slack by default

### PR DESCRIPTION
**Pull request by TomasMocek**

### CODE REVIEW

File: src/security_hub_reporter.py [hunk 1]
```diff
@@ -11,15 +11,19 @@
...
     findings_by_control_id = group_findings_by_control_id(findings)
     report, findings_count = build_findings_report(findings_by_control_id, ACCOUNT_ALIAS, ACCOUNT_ID)
 
-
-    send_report_to_sns(SNS_TOPIC_ARN, report)
+    if findings_count > 0:
+        send_report_to_sns(SNS_TOPIC_ARN, report)
+    elif PUBLISH_OK_MESSAGE_TO_SLACK == 'true':
+        ok_message = "Everything alright for {}".format(ACCOUNT_ALIAS)
+        send_report_to_sns(SNS_TOPIC_ARN, ok_message)
```
Comment (vladaGithub) [inline src/security_hub_reporter.py]: May I suggest?
```
 if findings_count > 0 or findings_count == 0 and PUBLISH_OK_MESSAGE_TO_SLACK == 'true':
        send_report_to_sns(SNS_TOPIC_ARN, report)
```

and in `def build_findings_report(by_control_id, account_alias, account_id):`
```
if findings_count == 0:
        report = "Everything alright for {}".format(ACCOUNT_ALIAS)

return [report, findings_count]
```

**Comment by neugeeug** [comment]
I went through all the stuff and LGTM. A few minor comments: 
1. Error handling.
If I see correctly, the lambda execution status will be always "Succeeded" regardless the real status.
Maybe we should spend a while to discuss a standard of error handling ?
One of possible way is rising an exception from the lambda handler in case that an important exception occurred somewhere in the Lambda body, what do you think ?  
2. I am wondering if the functionality of building metric should't be parameterized (default enabled) ?
Just in case somebody does not want to use DD and is going to use just simple Slack notifications.

File: src/security_hub_reporter.py (hunk 1 above)
Comment (TomasMocek) [reply src/security_hub_reporter.py]: Good suggestion, updated. Thanks1

**Comment by TomasMocek** [comment]
@neugeeug
1. Error handling could be definitely improved, but if Lambda fails at some point, it will fail with that exception..
2. I was wondering the same, but then I was thinking if there is really any harm in publishing the metric anyway. Price should be minimal. But I don't have any strong opinion, so I can add that possibility

**Review by vladaGithub — APPROVED** [review]

**Comment by neugeeug** [comment]
> @neugeeug
> 
> 1. Error handling could be definitely improved, but if Lambda fails at some point, it will fail with that exception..

As we discussed, not always will fail. You print the error message but the Lambda will finish with a success.

**Pull request merged**